nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,51 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reviews = require("./data/reviews")
+const reservations = require("./data/reservations")
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+app.get("/meals", async (request, response) => {
+  response.send(getMealsWithReviews())
+})
+
+app.get("/cheap-meals", async (request, response) => {
+  response.send(getMealsWithReviews().filter(meal => meal.price < 90))
+})
+
+app.get("/large-meals", async (request, response) => {
+  response.send(getMealsWithReviews().filter(meal => meal.price > 90))
```

Comment:
Large meals should : "Respond with the json for all the meals (including it's reviews) that can fit lots of people"
So instead of comparing the price, you should compare with the number of guests.